pkg/package-server/provider/inmem.go

```diff
@@ -248,16 +266,28 @@ func (m *InMemoryProvider) ListPackageManifests(namespace string) (*packagev1alp
 	return manifestList, nil
 }
 
-func (m *InMemoryProvider) GetPackageManifest(namespace, name string) (*packagev1alpha1.PackageManifest, error) {
-	m.mu.RLock()
-	defer m.mu.RUnlock()
-
-	var manifest packagev1alpha1.PackageManifest
-	for key, pm := range m.manifests {
-		if key.packageName == name && key.catalogSourceNamespace == namespace {
-			manifest = pm
+func (m *InMemoryProvider) WatchPackageManifests(namespace string, out chan packagev1alpha1.PackageManifest, stop <-chan struct{}) {
```

Comment:
I don't think this is being called anywhere, it looks like the `PackageManifestWatcher` is what's actually being used by the server